src/store/getters.js

```diff
@@ -14,9 +14,8 @@ export default {
   balanceCurrency({ current, balance }) {
     return (current.currencyRate * balance).toFixed(2);
   },
-  currentCurrency({ current }) {
-    return current.currency.toUpperCase();
-  },
+  formatCurrency: ({ current: { currency } }) => value =>
+    new Intl.NumberFormat('en', { style: 'currency', currency }).format(value),
```

Comment:
It even shows a ₽ sign only if Russian like locale is chosen. I don't think that we should implement it by ourselves.

Reply:
Does `en` should be hardcoded or we should change it dynamically based on the current locale settings?

Reply:
> should we change it dynamically based on the current locale settings?

I think so, is blocked by #470 

> As per specification Intl.NumberFormat uses locale strings as 'en-US', 'de-DE'

A Unicode BCP 47 locale identifier consists of
- a language code,
- (optionally) a script code,
- (optionally) a region (or country) code,
...

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_identification_and_negotiation

So, `en` is a correct BCP 47 locale identifier, that is used in `Intl.NumberFormat`